oap-server/analyzer/agent-analyzer/src/test/java/org/apache/skywalking/oap/server/analyzer/provider/meter/process/MeterProcessorTest.java

```diff
@@ -94,7 +94,7 @@ public void setup() throws StorageException, ModuleStartException {
     }
 
     @Test
-    public void testProcess() throws ModuleStartException {
```

Comment:
Iam confused, why does an exception statement matter for a UT?

Reply:
what does the UT mean?

Reply:
Unit test. I mean a method with exception should not break anything.

I want to know what do you want to fix.

Reply:
I just think the method does not have `throw ModuleStartException` possibility,so I think we can remove the `throws ModuleStartException` let code more clear